trellick/functions.php

```diff
@@ -7,7 +7,7 @@
  * @package trellick
  * @since trellick 1.0
  */
-declare( strict_types = 1 );
```

Comment:
I think these changes to Trellick should be kept. They're from this recent PR: https://github.com/Automattic/themes/pull/7570